apps/frontend/src/modules/events/components/Events.tsx

```diff
@@ -0,0 +1,94 @@
+import { EventCountableEdge, useEventsQuery } from "@/generated/graphql";
+import { Dialog, DialogContent, DialogTrigger, Header } from "@/ui";
+import { QuestionIcon } from "@/ui/icons";
+import PeopleIconLg from "@/ui/icons/PeopleIconLg";
+import { Table, TableBody, TableCell, TableHead, TableHeaderCell, TableRow } from "@tremor/react";
+import { format } from "date-fns";
+import { useState } from "react";
+
+export const Events = () => {
```

Comment:
@afolasope The Event component doesn't look completed. Are you still working on this PR?